app/build.gradle.kts

```diff
@@ -98,6 +98,8 @@ dependencies {
     implementation("androidx.navigation:navigation-compose:${LibVersion.navigationComposeVersion}")
     implementation("androidx.datastore:datastore-preferences:${LibVersion.dataStoreVersion}")
     implementation("androidx.paging:paging-compose:1.0.0-alpha16")
+    implementation("androidx.work:work-runtime-ktx:2.5.0")
+    implementation("androidx.compose.material:material:1.0.0-rc01")
```

Comment:
Why are we including `androidx.compose.material:material:1.0.0-rc01`?

Reply:
Yes , now I checked this has no use, but i will discard this branch after code review, I just pushed in remind for local and would remain in local.